static/app/components/events/interfaces/spans/measurementsPanel.tsx

```diff
@@ -92,26 +100,37 @@ const StyledLabelContainer = styled('div')`
   white-space: nowrap;
 `;
 
-const Label = styled('div')<{failedThreshold: boolean}>`
-  transform: translateX(-50%);
+const Label = styled('div')<{
+  failedThreshold: boolean;
+  isSingleLabel?: boolean;
+}>`
+  transform: ${p => (p.isSingleLabel ? `translate(-50%, 15%)` : `translateY(15%)`)};
   font-size: ${p => p.theme.fontSizeExtraSmall};
   font-weight: 600;
-  ${p => (p.failedThreshold ? `color: ${p.theme.red300};` : null)}
+  color: ${p => (p.failedThreshold ? `${p.theme.red300}` : `${p.theme.gray500}`)};
+  background: ${p => p.theme.white};
+  border: 1px solid;
+  border-color: ${p => (p.failedThreshold ? p.theme.red300 : p.theme.gray100)};
+  border-radius: 3px;
+  height: 75%;
+  display: flex;
```

Comment:
I'm seeing some magic numbers here, might be things we can use from theme (preferable), or check w/ Dora